README.md

```diff
@@ -39,12 +84,10 @@ This bot has been created using [Bot Framework](https://dev.botframework.com), i
   - Select `Bot.csproj` file
   - Press `F5` to run the project
 
-## Integration testing setup
-
-- Create a clockify account and generate the API key
+### Integration testing setup
```

Comment:
Before Integration testing setup, it will be nice to have instructions on how to run it locally. For example we use these properties: 
```json
{
  "LuisAppId": "",
  "LuisAPIKey": "",
  "LuisAPIHostName": "",
  "ProactiveBotApiKey": "",
  "KeyVaultName": "",
  "MicrosoftAppId": "",
  "MicrosoftAppPassword": ""
}
```

Locally app id and password can be left empty.